// react-material-navbar/dist/index.js
"use strict";
const React = require("react");
const Navbar = function (props) {
    const colorProps = {
        background: props.background,
        color: props.color,
    };
    return (React.createElement("nav", { className: `react-material-navbar`, style: Object.assign({}, props.style, colorProps) },
        React.createElement("div", { className: "react-material-navbar-left" }, props.leftAction),
        React.createElement("div", { className: `react-material-navbar-title` },
            React.createElement("span", null, props.title)),
        React.createElement("div", { className: "react-material-navbar-right" }, props.rightAction)));
};
exports.Navbar = Navbar;
const propTypes = {
    title: React.PropTypes.string.isRequired,
    color: React.PropTypes.string,
    background: React.PropTypes.string,
    leftAction: React.PropTypes.element,
    rightAction: React.PropTypes.element,
    style: React.PropTypes.object,
    key: React.PropTypes.any,
    ref: React.PropTypes.any,
};
const defaultProps = {
    color: "#ffffff",
    background: "#0d47a1",
};
Navbar.propTypes = propTypes;
Navbar.defaultProps = defaultProps;
Object.defineProperty(exports, "__esModule", { value: true });
exports.default = Navbar;
